projects/notebooks/testing/config.yaml

```
@@ -213,10 +215,10 @@ ldap:
 
 rhods:
   catalog:
-    image: brew.registry.redhat.io/rh-osbs/iib
-    tag: 752361
+    image: registry-proxy.engineering.redhat.com/rh-osbs/iib
```

you need to change this back to `brew.registry.redhat.io/rh-osbs/iib`, this one won't work

```suggestion
    image: brew.registry.redhat.io/rh-osbs/iib
```